.shellcheckrc

```diff
@@ -0,0 +1,9 @@
+# rules currently disabled:
+#
+#   SC1091 (info): Not following: <sourced file> was not specified as input (see shellcheck -x)
+#   SC2004 (style): $/${} is unnecessary on arithmetic variables.
+#   SC2129 (style): Consider using { cmd1; cmd2; } >> file instead of individual redirects.
+#   SC2155 (warning): Declare and assign separately to avoid masking return values.
+#   SC2164 (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
+#
+disable=SC1091,SC2004,SC2129,SC2155,SC2164
```

Comment:
Why is this file required?

Reply:
This is inherited from vllm and for the shellcheck test of `format.sh`. If it is disabled, some additional warning appear when running `format.sh`. 
In the vllm-spyre plugin case, only `SC2155` warnings appear, so I can remove the others if you think it is better.

Reply:
No, let's keep it the same as vllm. Makes sense.